plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/FailoverEndpointDeserializer.java

```diff
@@ -67,7 +68,9 @@ public FailoverEndPoint createNode(IGraphicalEditPart part,AbstractEndpoint obje
 		if(StringUtils.isNotBlank(failoverEndpoint.getName())){
              executeSetValueCommand(END_POINT__END_POINT_NAME, failoverEndpoint.getName());
 		}
-		
+
```

Comment:
Can you please remove this empty line here and line 73?

Reply:
Fixed